docs/source/services/index.rst

```diff
@@ -3,12 +3,16 @@ Services
 ========
 
 AtlasDB relies on three distinct services. They are the Key Value Service, the Lock Service, and the Timestamp Service.
-Each of the services is described in detail below.  The intention of this documentation is to provide a high level overview of the architecture of
-each of the services as well as the guarentees they uphold.
+Each of the services is described in detail below.  The intention of this documentation is to provide a high level
+overview of the architecture of each of the services as well as the guarantees they uphold.
+
+In the past, AtlasDB would typically run an embedded timestamp and lock service. We are currently developing an
```

Comment:
It's not the past yet! I think this would be better as "AtlasDB typically runs ... however, we are currently developing an external time and lock server. Please consult <direct link to docs>"